Source/relay/TourGuide/Quests/Level 11 - Hidden City.ash

```diff
@@ -315,7 +315,13 @@ void QLevel11HiddenCityGenerateTasks(ChecklistEntry [int] task_entries, Checklis
                 
                 int totalTurnsSpent = $location[the hidden office building].turns_spent;
                 
-                int delayForNextNoncombat = 4 - (totalTurnsSpent - 1) % 5;
+                int delayForNextNoncombat;
+
+                if (totalTurnsSpent < 6) {
+                    delayForNextNoncombat = 5 - totalTurnsSpent;
+                } else {
+                    delayForNextNoncombat = 4 - (totalTurnsSpent - 1) % 5;
+                }
```

Comment:
```suggestion
                int delayForNextNoncombat = 4 - (totalTurnsSpent - 1) % 5;;

                if (totalTurnsSpent < 6) {
                    delayForNextNoncombat = 5 - totalTurnsSpent;
                }
```
This is the professional C++ dev in me speaking. Always initialise your variables and the fewer branches the better. ASH/Java likely won't care about the tiny amount of performance from pruning branches but these are good habits generally across languages.

Reply:
appreciate the general advice and performance considerations for my curiosity,

for C I get the impression that is at the same time a good rule of thumb and wrong
also in this case I would wonder how that branch cost compares to assigning that variable two times, or in ash just doing the operation two times

but anyway the fix only changes value in one case where this location was set with 0 turns spent, so is something like this better
>int delayForNextNoncombat = (totalTurnsSpent == 0) ? 5 : (4 - (totalTurnsSpent - 1) % 5);